docs/governance/how-to-vote.md

```diff
@@ -1,19 +1,20 @@
-# <img class="dcr-icon" src="/img/dcr-icons/TicketVoted.svg" /> **How To Vote**
+# <img class="dcr-icon" src="/img/dcr-icons/TicketVoted.svg" /> **How to Vote on Rule Change Proposals**
```

Comment:
Please update this to: 

```
How to Vote on consensus rule changes
```

So it matches its entry in the navigation menu

Reply:
Have implemented change. However, I noticed that all pages except governance pages capitalize all words in the page title. Have opened Issue to fix: https://github.com/decred/dcrdocs/issues/714